dubbo-registry/dubbo-registry-api/src/test/java/org/apache/dubbo/registry/support/AbstractRegistryTest.java

```diff
@@ -48,6 +49,8 @@
     @Before
     public void init() {
         URL url = URL.valueOf("dubbo://" + NetUtils.getLocalAddress().getHostName() + ":2233");
+        //sync update cache file
+        url = url.addParameter( "save.file",true );
```

Comment:
not the same code style.
`use ("save.file", true);`

Reply:
ok